File: week6/sentimental-mario-more/mario.py
while True:
    height = input("Give me a number between 1 and 8: ")
    try:
        height_int = int(height)
        if height_int > 0 and height_int < 9:
            break
    except ValueError:
        pass


for i in range(height_int - 1, -1, -1):
    if i == 0:
        print("#" * (height_int - i), end="")
    else:
        print(" " * i + "#" * (height_int - i), end="")
    print("  ", end="")
    print("#" * (height_int - i))

File: Week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int input;
    do
    {
        input = get_int("Write a number between 1 and 8: ");
    }
    while (input < 1 || input > 8);

    for (int i = 0; i < input; i++)
    {
        for (int s = 1; s < input - i; s++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}